src/main/java/core/basesyntax/operations/BalanceOperation.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
+    private final Storage storage;

```

Comment:
The 'Storage' should not be directly used here. Instead, follow the encapsulation principle and use a service to interact with the storage. This will make your 'BalanceOperation' class independent and reusable.